examples/multiplatform/tvOSApp/BUILD

```diff
@@ -1,80 +1,15 @@
-load("@build_bazel_rules_apple//apple:apple.bzl", "local_provisioning_profile")
-load("@build_bazel_rules_apple//apple:resources.bzl", "apple_resource_group")
-load("@build_bazel_rules_apple//apple:tvos.bzl", "tvos_application")
-load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
-load(
-    "@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:experimental.bzl",
-    "xcode_provisioning_profile",
-)
-load(
-    "//examples/multiplatform:xcodeproj_targets.bzl",
-    "TEAMID",
-    "TVOS_BUNDLE_ID",
-)
+load("//xcodeproj:xcodeproj.bzl", "xcodeproj")
+load(":xcodeproj_targets.bzl", "XCODEPROJ_TARGETS")
 
-config_setting(
-    name = "release_build",
-    values = {
-        "compilation_mode": "opt",
-    },
-)
-
-config_setting(
-    name = "device_build",
-    values = {
-        "cpu": "tvos_arm64",
-    },
+xcodeproj(
+    name = "xcodeproj",
+    project_name = "tvOS App",
+    tags = ["manual"],
+    top_level_targets = XCODEPROJ_TARGETS,
 )
```

Comment:
Shouldn't need this. It should just be part of the grander `multiplatform` project now.